space/authz/authz.go

```diff
@@ -3,18 +3,17 @@ package authz
 
 import (
 	"context"
-	"errors"
-	"fmt"
 	"net/http"
 	"time"
 
 	"github.com/almighty/almighty-core/application"
 	"github.com/almighty/almighty-core/auth"
-	errs "github.com/almighty/almighty-core/errors"
+	"github.com/almighty/almighty-core/errors"
```

Comment:
Good change! I'm glad we can align the package name/alias use across all files., i.e., using `errs` for `github.com/pkg/errors` and `errors` for your own package.